### PR TITLE
Change the test deployment ID format

### PR DESCRIPTION
**Pull request by simonxiaoss**
Now the Azure resource group created by LISAv2 will be something like this:
"LISAv2-SingleVM-simonxiaoss-AX30-20181212140326".

### CODE REVIEW

File: Libraries/Azure.psm1 [hunk 1]
```diff
@@ -2024,10 +2024,10 @@ Function Create-RGDeploymentWithTempParameters([string]$RGName, $TemplateFile, $
 
 Function Create-AllRGDeploymentsWithTempParameters($templateName, $location, $TemplateFile, $TemplateParameterFile) {
     $resourceGroupCount = 0
-    $curtime = Get-Date
     $isServiceDeployed = "False"
     $retryDeployment = 0
-    $groupName = "ICA-RG-" + $templateName + "-" + $curtime.Month + "-" + $curtime.Day + "-" + $curtime.Hour + "-" + $curtime.Minute + "-" + $curtime.Second
+    $uniqueId = New-TimeBasedUniqueId
+    $groupName = "ICA-RG-" + $templateName + "-" + $uniqueId
```
Comment (lizzha) [inline Libraries/Azure.psm1]: "LISAv2-"?

File: Libraries/Azure.psm1 [hunk 2]
```diff
@@ -380,14 +380,14 @@ Function Create-AllResourceGroupDeployments($setupType, $xmlConfig, $Distro, [st
                 }
             }
             if ($readyToDeploy) {
-                $curtime = ([string]((Get-Date).Ticks / 1000000)).Split(".")[0]
+                $uniqueId = New-TimeBasedUniqueId
                 $isServiceDeployed = "False"
                 $retryDeployment = 0
                 if ( $null -ne $RG.Tag ) {
-                    $groupName = "ICA-RG-" + $RG.Tag + "-" + $Distro + "-" + "$TestID-" + "$curtime"
```
Comment (ader1990) [inline Libraries/Azure.psm1]: this constant (aka $AZURE_RESOURCE_PREFIX) should be defined only once at the beginning of the code, to be easily changed in the future -- or parameterizable.

File: Libraries/TestHelpers.psm1 [hunk 3]
```diff
@@ -0,0 +1,36 @@
+##############################################################################################
```
Comment (ader1990) [inline Libraries/TestHelpers.psm1]: why do you need to create another module? we already have commonfunctions.psm1 or LISAv2-Framework.psm1 for the same purpose.

File: Libraries/TestHelpers.psm1 [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+.EXAMPLE
+
+
+#>
+###############################################################################################
+
+Function New-TestID()
+{
+	return "{0}{1}" -f $(-join ((65..90) | Get-Random -Count 2 | ForEach-Object {[char]$_})), $(Get-Random -Maximum 99 -Minimum 11)
```
Comment (ader1990) [inline Libraries/TestHelpers.psm1]: line too long, please try to keep the line length to a reviewable length

File: Run-LisaV2.ps1 [hunk 5]
```diff
@@ -113,8 +113,8 @@ try {
 
 	# Prepare log folder
 	$LogDir = Join-Path $WorkingDirectory "TestResults\$(Get-Date -Format 'yyyy-dd-MM-HH-mm-ss-ffff')"
-	Set-Variable -Name LogDir      -Value $LogDir      -Scope Global -Force
-	New-Item -ItemType Directory -Path $LogDir -Force | Out-Null
+	Set-Variable -Name LogDir        -Value $LogDir -Scope Global -Force
```
Comment (ader1990) [inline Run-LisaV2.ps1]: this line contains unnecessary spaces

File: Run-LisaV2.ps1 [hunk 6]
```diff
@@ -113,8 +113,8 @@ try {
...
+	New-Item     -ItemType Directory -Path $LogDir  -Force | Out-Null
```
Comment (ader1990) [inline Run-LisaV2.ps1]: ditto

File: Libraries/Azure.psm1 (hunk 1 above)
Comment (simonxiaoss) [reply Libraries/Azure.psm1]: Fixed

**Comment by ader1990** [comment]
@simonxiaoss I see that none of my review comments have been addressed, not even the code formatting ones. is there a reason you decided to forcefully approve my requests without fixing the code?

**Pull request merged**